### PR TITLE
feat(sdk): add GetTransactions + export more types

### PR DESCRIPTION
**Pull request by kevzzsk**
#### What this PR does / why we need it:

1. export more typing
2. add `GetTransactions` RPC
3. update `GetInscriptions` RPC to include `includePostage` params

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] No console errors 
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: packages/sdk/src/api/types.ts [hunk 1]
```diff
@@ -19,14 +19,23 @@ export interface GetUnspentsResponse {
   unspendableUTXOs: UTXO[]
 }
 
-export interface GetTxOptions {
+export interface GetTransactionOption {
```
Comment (danadi7) [inline packages/sdk/src/api/types.ts:22]: Should we keep it the `s` in `Options` for consistency?

File: packages/sdk/src/api/types.ts [hunk 2]
```diff
@@ -19,14 +19,23 @@ export interface GetUnspentsResponse {
...
   txId: string
   ordinals?: boolean
   hex?: boolean
   witness?: boolean
   decodeMetadata?: boolean
 }
 
+export interface GetTransactionsOption {
```
Comment (danadi7) [inline packages/sdk/src/api/types.ts:30]: Here too

**Review by joundy — APPROVED** [review]
LGTM for the RPC request

**Review by danadi7 — APPROVED** [review]
LGTM!

File: packages/sdk/src/transactions/types.ts [hunk 3]
```diff
@@ -39,12 +42,42 @@ export type Transaction = {
...
+  vin: Vin[]
+  vout: Vout[]
+  blockHash: string
+  blockHeight: number
+  blockTime: number
+  confirmations: number
+  time: number
+  weight: number
+  fee: number
+}
```
Comment (Nanosync) [inline packages/sdk/src/transactions/types.ts:71]:
```suggestion
export type TransactionV2 = Omit<Transaction, 'blockhash' | 'blockheight' | 'blocktime'> & {
  blockHash: string
  blockHeight: number
  blockTime: number
}
```

**Review by Nanosync — APPROVED** [review]

**Pull request merged**